package.json

```
@@ -40,7 +47,8 @@
         "@zenflux/eslint": "latest",
         "@zenflux/jest": "latest",
         "eslint": "latest",
-        "typescript": "latest"
+        "typescript": "latest",
```

```suggestion
        "typescript": "latest",
``` remove from other packages then